README.md

```diff
@@ -246,5 +247,33 @@ to report public suggestions or bugs.
 We chat in [our Mattermost instance](https://chat.slint-ui.com) where you are
 welcome to listen in or ask your questions.
 
-You can of course also tweet at [@slint-ui](https://twitter.com/slint_ui) or
-contact us privately via email to [info@slint-ui.com](mailto://info@slint-ui.com).
+You can of course also contact us privately via email to [info@slint-ui.com](mailto://info@slint-ui.com).
+
+[#1]: https://www.youtube.com/watch?v=_BDbNHrjK7g
+[#2]: https://www.youtube.com/watch?v=NNNOJJsOAis
+[#3]: https://www.youtube.com/watch?v=dkBwNocItGs
+[#4]: https://slint-ui.com/resources/gallery_win_screenshot.png "Gallery"
+[#5]: https://slint-ui.com/resources/gallery_mac_screenshot.png "Gallery"
+[#6]: https://slint-ui.com/resources/gallery_linux_screenshot.png "Gallery"
+[#7]: https://slint-ui.com/resources/printerdemo_screenshot.png "Printer Demo"
+[#8]: https://slint-ui.com/demos/printerdemo/
+[#9]: https://slint-ui.com/resources/puzzle_screenshot.png "Slide Puzzle"
+[#10]: https://slint-ui.com/demos/slide_puzzle/
+[#11]: https://slint-ui.com/resources/energy-monitor-screenshot.png "Energy Monitor Demo"
+[#12]: https://slint-ui.com/demos/energy-monitor/
+[#13]: https://slint-ui.com/resources/gallery_screenshot.png "Gallery Demo"
+[#14]: https://slint-ui.com/demos/gallery/
+[#15]: https://slint-ui.com/docs/cpp
+[#16]: https://slint-ui.com/docs/tutorial/cpp
+[#17]: https://github.com/slint-ui/slint-cpp-template
+[def]: https://img.shields.io/crates/v/slint
```

Comment:
I'm curious, why is this `def` instead of `18`? :)

Reply:
This was auto inserted by VSCode. Not sure why it did that.. it looks like it maintained the numbering for others. Changed it to #18